xrpl/asyncio/ledger/main.py

```diff
@@ -46,21 +48,44 @@ async def get_latest_open_ledger_sequence(client: Client) -> int:
     raise XRPLRequestFailureException(response.result)
 
 
-async def get_fee(client: Client) -> str:
+async def get_fee(
+    client: Client, *, max_fee: Optional[float] = 2, fee_type: str = "open"
+) -> str:
     """
     Query the ledger for the current minimum transaction fee.
 
     Args:
         client: the network client used to make network calls.
+        max_fee: The maximum fee in XRP that the user wants to pay. If load gets too
+            high, then the fees will not scale past the maximum fee. If None, there is
+            no ceiling for the fee. The default is 2 XRP.
+        fee_type: The type of fee to return. The options are "open" (the load-scaled
+            fee to get into the open ledger) or "minimum" (the minimum transaction
+            cost). The default is "open".
 
     Returns:
-        The minimum fee for transactions.
+        The transaction cost, in drops.
 
     Raises:
+        XRPLException: if an incorrect option for `fee_type` is passed in.
         XRPLRequestFailureException: if the rippled API call fails.
     """
     response = await client.request_impl(Fee())
     if response.is_successful():
-        return cast(str, response.result["drops"]["minimum_fee"])
+        result = response.result["drops"]
+        if fee_type == "open":
+            fee = cast(str, result["open_ledger_fee"])
+        elif fee_type == "minimum":
+            fee = cast(str, result["minimum_fee"])
```

Comment:
Why do you use `cast(str,...)` here instead of `str(...)`?

Reply:
It's already a string, but mypy doesn't know that - `cast` is the same as `as string` in TS.